### ISSUE TITLE
Feature: set default channel for certain type of users

### ISSUE THREAD

**Issue by 1stposition-ad** [RocketChat/Rocket.Chat#7026]
A cool new feature would be to set default channels for certain type of users.
For e.g that all admins can join automatically one or more admin channels, external users another one, and so on.

**Comment by ckiltolier** [comment]
Yeah, like for an international chat, it would be great that people arrives on their country channel !

**Comment by rodrigok** [comment]
Related to https://github.com/RocketChat/Rocket.Chat/issues/8005
